WNPRC_EHR/resources/etls/arrow_to_ehr_protocol_merge.xml

```diff
@@ -0,0 +1,32 @@
+<etl xmlns="http://labkey.org/etl/xml">
+    <name>Arrow ETL 2: Merge wnprc protocol copy to ehr protocol table</name>
+    <description>Merges data from wnprc.arrow_protocols into ehr.protocol table</description>
+    <transforms>
+        <transform id="step1">
+            <description>Copy to target</description>
+            <source queryName="MaxSpeciesDistinct" schemaName="wnprc"/>
+            <destination queryName="protocol" schemaName="ehr" targetOption="merge">
+                <columnTransforms>
+                    <column source="protocol_id" target="protocol"/>
+                    <column source="protocol_title" target="title"/>
+                    <column source="pi_name" target="inves"/>
+                    <column source="date_approved" target="approve"/>
+                    <column source="date_expiration" target="enddate"/>
+                    <column source="sum_three_yr" target="maxAnimals"/>
+                    <column source="usda_code" target="usda_level"/>
+                    <column source="protocol_id" target="pdf"/>
```

Comment:
I don't know anything about the data setup, but this seemed strange to me. Is this right? protocol_id -> pdf

Reply:
yeah it's a bit strange, the pdf column is the protocol number and it's configured to point to a file in the table customizer: https://github.com/WNPRC-EHR-Services/wnprc-modules/blob/06573469157b77ebea82e3c8cda0d57ac0cb7c8a/WNPRC_EHR/src/org/labkey/wnprc_ehr/table/WNPRC_EHRCustomizer.java#L904